test/suites/container_devices_nic_bridged_filtering.sh

```diff
@@ -279,6 +314,8 @@ test_container_devices_nic_bridged_filtering() {
   lxc config device set "${ctPrefix}A" eth0 ipv6.address 2001:db8::2
   lxc config device set "${ctPrefix}A" eth0 security.mac_filtering false
   lxc config device set "${ctPrefix}A" eth0 security.ipv6_filtering true
+  lxc config device set "${ctPrefix}A" eth0 ipv6.routes 2001:db8:0:0:1::/96
```

Comment:
Not a problem per se, but interested to know the reason why you chose a /96 here?

Reply:
This was for the same reason as https://github.com/lxc/lxd/pull/9953/files#r821595540. The IPv6 subnet is `2001:db8::/64` (https://github.com/lxc/lxd/pull/9953/files#diff-1f0a251e7be7b83cffcce0dca5ce41675fefc0fbb926d78a56f7234c77b685edR39) so I wanted to define the routes within that range so that I can assign it an IP address within that range for testing later. 

Without doing this, I was getting "Network unreachable" from the ping.